sdk/communication/azure-communication-callautomation/swagger/README.md

```diff
@@ -148,6 +148,8 @@ directive:
 - remove-model: PlayCompleted
 - remove-model: PlayFailed
 - remove-model: ResultInfo
+- remove-model: RecognizeCompleted
```

Comment:
Yeah, this is what I was talking about removing the events from being autogenerated. 
I think that you need to run auto rest again or delete the events manually.